doc/nrf/ug_thread_tools.rst

```diff
@@ -37,22 +37,242 @@ Thread Border Router
+
+            .. code-block:: console
+
+               JLinkExe -device NRF52840_XXAA -if SWD -speed 4000 -autoconnect 1 -SelectEmuBySN *SEGGER_ID*
+               J-Link>MSDDisable
+               Probe configured successfully.
+               J-Link>exit
+
+            Replace *SEGGER_ID* with the SEGGER ID of your nRF52840 Development Kit.
+            This setting remains valid even if you flash another firmware onto the device.
+         #. Power-cycle the device to apply the changes.
+
+Configuring using Docker
+========================
+
+For the development purposes, you can run the OpenThread Border Router on any Linux-based system using a Docker container that already has the Border Router installed.
+This solution is better when you are only interested in direct communication between your Border Router and the Thread network.
+For example, you can use the Docker container when you want to establish IP communication between an application running on Linux (such as Python Controller for Matter) and an application running on a Thread node.
+
+If you go for the Docker container option, make sure to :ref:`configure radio co-processor <ug_thread_tools_tbr_rcp>` first.
+
+To install and configure OpenThread Border Router on any Linux-based operating system using the Docker container, complete the following steps:
+
+#. Install the Docker daemon:
+
+   .. code-block:: console
+
+      sudo apt update && sudo apt install docker.io
+
+#. Start the Docker daemon:
+
+   .. code-block:: console
+
+      sudo systemctl start docker
+
+#. Create an IPv6 network for the OpenThread Border Router container in Docker:
+
+   .. code-block:: console
+
+      sudo docker network create --ipv6 --subnet fd11:db8:1::/64 -o com.docker.network.bridge.name=otbr0 otbr
+
+#. Download the latest version of the OpenThread Border Router Docker image by running the following command:
+
+   .. code-block:: console
+
+      docker pull openthread/otbr
+
+#. Connect the radio co-processor that you configured in :ref:`ug_thread_tools_tbr_rcp` to the Border Router device.
+#. Start the OpenThread Border Router container using the following command (in the last line, replace */dev/ttyACM0* with the device node name of the OpenThread radio co-processor):
+
+   .. parsed-literal::
+      :class: highlight
+
+      sudo docker run -it --rm --privileged --name otbr --network otbr -p 8080:80 \
+      --sysctl "net.ipv6.conf.all.disable_ipv6=0 net.ipv4.conf.all.forwarding=1 net.ipv6.conf.all.forwarding=1" \
+      --volume */dev/ttyACM0*:/dev/radio openthread/otbr --radio-url spinel+hdlc+uart:///dev/radio
+
+#. Form the Thread network using one of the following options:
+
+   * Follow the instruction in the `OpenThread Border Router Codelab tutorial step 2`_.
+   * Open the ``http://*localhost*:8080/`` address in a web browser and choose :guilabel:`Form` from the menu.
+
```

Comment:
If we need to use On-Mesh Prefix set in this step later on, maybe it would be good idea to mention it here, to not force User to go back again. How about adding the following sentence: "Please note the selected On-Mesh Prefix value, as in further step it will be needed in order to configure routing between the networks."